test/exe/Main.hs

```diff
@@ -1243,7 +1243,7 @@ completionTests
         compls <- getCompletions docId (Position 1 7)
         liftIO $ map dropDocs compls @?= 
           [complItem "head" (Just CiFunction) (Just "[a] -> a")
-                     "*Defined in 'Prelude'*\n*\t*\t*\n\n\nExtract the first element of a list, which must be non-empty."]
```

Comment:
This doesn’t actually check anything about the docs. The “Defined in Prelude” part is completely independent of this PR (although this PR slightly changes the wording). Let’s please check that the docs actually work. Some CPP to not check this on older GHCs is totally fine.

Reply:
New tests are in!